L/LLVM/common.jl

```diff
@@ -105,6 +123,9 @@ fi
 if [[ "${LLVM_MAJ_VER}" -gt "12" ]]; then
     ninja -j${nproc} mlir-linalg-ods-yaml-gen
 fi
+if [[ "${LLVM_MAJ_VER}" -gt "14" ]]; then
```

Comment:
Is this correct to reference version 14?  Should it be version 15?  Same for a similar block added below.

Reply:
Valentin pointed out that this is `-gt`, not `-ge`, so this is correct!